pandas/tests/groupby/test_allowlist.py

```diff
@@ -309,6 +309,7 @@ def test_tab_completion(mframe):
         "rank",
         "cumprod",
         "tail",
+        "rows",
```

Comment:
same comment as above (IOW remove for now)

Reply:
Done